.drone.yml

```diff
@@ -18,7 +18,7 @@ steps:
     commands:
       - yarn lint
 
-  - name: Type check
```

Comment:
lol

Reply:
You're still pissed about this?

Reply:
No, not now. At least @eikhr is a cultured man

Reply:
Sorry, I read the situation wrong,
<img width="376" alt="image" src="https://user-images.githubusercontent.com/23152018/181626990-2dba88c4-e7c0-42a0-b2ea-fab5b1568359.png">
From my pov you still complained about the "Type check", but I notice that you referred to the fix

I rest my case

Mic drop